__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed.js");
+const categories = require("../db/data/test-data/categories.js");
+require("jest-sorted");

```

Comment:
This line isn't doing anything so should be removed

Reply:
line 7?